src/mahoji/commands/clue.ts

```diff
@@ -155,6 +138,30 @@ export const clueCommand: OSBMahojiCommand = {
 		const hasOrnateJewelleryBox = poh.jewellery_box === getPOHObject('Ornate jewellery box').id;
 		const hasJewelleryBox = poh.jewellery_box !== null;
 		const hasXericTalisman = poh.amulet === getPOHObject("Mounted xeric's talisman").id;
+		const hasAchievementDiaryCape = user.hasEquippedOrInBank('Achievement diary cape');
+
+		// Stash Unit boost
+		const stashBoost = await getStashBoost(userID, clueTier.name);
+		boosts.push(`${stashBoost.toFixed(2)}% for built STASH Units`);
+		duration *= 1 - stashBoost / 100;
```

Comment:
Use our standard way instead: `duration = reduceNumByPercent(duration, percentage)`

Reply:
ACTUALLY - if this number can be negative, ignore this.